raiwidgets/tests/test_model_analysis_dashboard_input.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+from sklearn.model_selection import train_test_split
+import shap
+import sklearn
+import mock
+from responsibleai import ModelAnalysis
+from raiwidgets.model_analysis_dashboard_input import \
```

Comment:
Will customers ever import `ModelAnalysisDashboardInput`? If not let's make the module private

Reply:
probably in separate pr, all other dashboards are like this as well.